.github/workflows/coverage.yml

```diff
@@ -35,12 +35,14 @@ jobs:
           pip install -r requirements.txt
           dbt deps
       - name: Initialize Open edX
-        continue-on-error: true
+        continue-on-error: false
         run: |
           tutor plugins enable aspects
           tutor config save
           tutor local start -d
```

Comment:
@bmtcril you need only to start the clickhouse container for this to work

Reply:
I think I'll also need to change the load test config to talk directly to clickhouse instead of Ralph, too.

Reply:
Or you can start both